lighthouse-cli/bin.ts

```diff
@@ -143,6 +147,10 @@ if (!cliFlags.outputPath && cliFlags['output-path']) {
   cliFlags.outputPath = cliFlags['output-path'];
 }
 
+if (!cliFlags.maxWaitForLoad && cliFlags['max-wait-for-load']) {
```

Comment:
What was the google3 dependency on yargs that kept us here, are they game to update?